src/deepsparse/benchmark_model/benchmark_model.py

```diff
@@ -41,9 +41,11 @@
                         The number of physical cores to run the analysis on,
                         defaults to all physical cores available on the system
   -s {async,sync}, --scenario {async,sync}
-                        Choose between using sync/async scenarios. This is
-                        similar to the single-stream/multi-stream scenarios.
-                        The default value is async.
+                        Choose between using the async, sync and elastic scenarios.
+                        Sync and async are similar to the single-stream/multi-stream
+                        scenarios. Elastic is a newer scenario that behaves similarly
+                        to the async scenario but uses a different scheduling backend.
+                        Default value is async.
```

Comment:
The description of elastic seems too vague to me.  Like why would anyone choose it over sync/async?   Diff looks fine otherwise.